database/migrations/postgres/1733415120086267_deployments_autoincrement.up.sql

```diff
@@ -0,0 +1,30 @@
+ALTER TABLE IF EXISTS deployments ADD COLUMN IF NOT EXISTS version INTEGER;
+DO $$ 
+BEGIN
+    IF EXISTS (SELECT 1 
+               FROM information_schema.columns 
+               WHERE table_name = 'deployments' 
+                 AND column_name = 'eslversion') THEN
+        EXECUTE 'WITH ordered_rows AS (
+            SELECT eslversion, envname, appname, ROW_NUMBER() OVER (ORDER BY eslversion) AS row_num
+            FROM deployments
+        )
+        UPDATE deployments
+        SET version = ordered_rows.row_num
+        FROM ordered_rows
+        WHERE deployments.eslversion = ordered_rows.eslversion AND deployments.appname = ordered_rows.appname AND deployments.envname = ordered_rows.envname;';
```

Comment:
are we sure this updates 100% of the columns?

Reply:
Yes, because (eslversion, appname, envame) is the primary kkey for deployments table.